server/server.go

```diff
@@ -1400,3 +1404,49 @@ func (server *HTTPServer) readListOfDisabledRulesForClusters(
 
 	return response.DisabledRules, nil
 }
+
+// Method readListOfClusterDisabledRules returns user disabled rules for given cluster list
```

Comment:
Mismatch between the comment and the method name

Reply:
thank you, good catch